Description: Remove embedded jsoncpp, only used in tests that aren't built.
Author: Felix Geyer <fgeyer@debian.org>
Last-Update: 2014-01-16

--- a/lib/phusion_passenger/common_library.rb
+++ b/lib/phusion_passenger/common_library.rb
@@ -437,13 +437,6 @@
 			Utils/fib.h
 			Utils/fibpriv.h
 		)
-	define_component 'Utils/jsoncpp.o',
-		:source   => 'Utils/jsoncpp.cpp',
-		:category => :other,
-		:deps     => %w(
-			Utils/json.h
-			Utils/json-forwards.h
-		)
 
 	#'BCrypt.o' => %w(
 	#	BCrypt.cpp
